### ISSUE TITLE
Backup in another namespace

### ISSUE THREAD

**Issue by fmonorchio** [stashed/stash#1261]
Hi all,

I want to use Stask as my Backup/Recovery solution. There is the possibility to create the secret/repo/backupConfiguration resources in the stash namespace and use these for deployment in other namespaces? i don't want to sync the secret with s3 credentials in all the namespaces that I have.

Thanks

**Comment by hossainemruz** [comment]
No. We don't allow the backup job / sidecar to access any resources in other namespaces. This is for security purposes. Otherwise, a user will be able to violate cluster RBAC using such capabilities.

**Comment by Rojikku** [comment]
I should be able to specify repository.namespace in order to avoid re-creating the repository in every namespace of interest.
This is most probably the largest issue I've come across with Stash.

**Comment by hossainemruz** [comment]
This feature already has been implemented in master branch. It will be available in the next release.
